op/model.py

```diff
@@ -72,6 +71,38 @@ def __init__(self, name, backend, cache):
 
         self._is_our_app = self.name == self._backend.app_name
 
+        self._status = None
```

Comment:
The lines above may all be packed without spaces in between. It'll probably look nicer.